addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -21,79 +21,42 @@
             padding: 0!important;
         }
         background-color: $o-view-background-color;
-        width: auto;  // bootstrap override
 
         // Inform the user that he may click on a cell to be redirected to a list view of the
         // items corresponding to the clicked cell
         &.o_enable_linking {
             .o_pivot_cell_value:not(.o_empty):hover {
                 color: $o-brand-primary;
                 cursor: pointer;
-
             }
         }
 
-        thead th:not(.o_pivot_header_cell_closed):not(.o_pivot_header_cell_opened):not(.o_pivot_header_cell) {
-            text-align: center;
-        }
-
         th, td {
-            border-color: gray('200');
-            border-width: $border-width $border-width $border-width $border-width;
+            border-color: gray('300');
+            border-width: $border-width;
```

Comment:
Can't think about an odoo's table that requires bootstrap native the double border... check if we can set i globally

Reply:
It will be possible after the migration to BS 5, I let a comment.